integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMaxConcurOptions.java

```diff
@@ -0,0 +1,760 @@
+// Copyright (c) 2022, Oracle and/or its affiliates.
```

Comment:
```suggestion
// Copyright (c) 2023, Oracle and/or its affiliates.
```

Reply:
You need to update the copyright date on the other two files also.